R/pkg/R/functions.R

```diff
@@ -3083,11 +3011,10 @@ setMethod("rank",
             column(jc)
           })
 
-# Expose rank() in the R base package
-#' @param x a numeric, complex, character or logical vector.
-#' @param ... additional argument(s) passed to the method.
-#' @name rank
-#' @rdname rank
+#' @details
+#' \code{rank}: Exposes \code{rank()} in the R base package. In this case, \code{x}
```

Comment:
I'm actually mixed in whether we should group the doc for this.
I guess our hands are tied and we have to have documentation for it